game/models.py

```diff
@@ -36,6 +36,7 @@
 # identified as the original program.
 from django.contrib.auth.models import User
 from django.db import models
+from django.utils.translation import ugettext_lazy
```

Comment:
Do these need to be lazy? The text is in the `__unicode__` message, so it's unlikely to be evaluated at initialisation time?